GoogleForCreators/web-stories-wp
# Consecutive text placement

## Summary

Adds logic for consecutively placing text presets.


## Relevant Technical Choices
- Positions elements under each other when added consecutively.
- If there is any change made between adding the presets, the position will be reset, too.


## To-do




## User-facing changes

Text presets are positioned if clicked on consecutively:
![presets](https://user-images.githubusercontent.com/3294597/91180160-b9f69400-e6e7-11ea-8503-2c0e949a145f.gif)



## Testing Instructions
I
1. Add different text presets consecutively.
2. Verify that if presets are added right after each other, the next preset is positioned below the previous.
3. Verify that the preset doesn't go out of the bottom of the page, staggering starts from the default position (vertically centered) if the room ends up in the bottom of the page.
4. Verify that if any change is made between adding the presets, the preset is added to the default position (the consecutiveness counts only if the presets are added right after each other).



---



Fixes #1924
